ethpm/utils/deployments.py

```diff
@@ -0,0 +1,97 @@
+from typing import Any, Dict, Generator, List, Tuple
+
+from eth_utils import to_bytes, to_canonical_address, to_tuple
+from web3 import Web3
+
+from ethpm.exceptions import BytecodeLinkingError, ValidationError
+
+
+def get_linked_deployments(deployments: Dict[str, Any]) -> Dict[str, Any]:
+    """
+    Returns all deployments found in a chain URI's deployment data that contain link dependencies.
+    """
+    linked_deployments = {
+        dep: data
+        for dep, data in deployments.items()
+        if "runtime_bytecode" in data
+        and "link_dependencies" in data["runtime_bytecode"]
+    }
+    for deployment, data in linked_deployments.items():
+        if deployment == data["runtime_bytecode"]["link_dependencies"][0]["value"]:
```

Comment:
The `[0]` indexing here confuses me as well.  Why do we have a sequence style data structure which only has one element?